config/settings.yml

```diff
@@ -921,8 +921,20 @@
     - MoveIntoResourcePool
     :VmSuspendedEvent:
     - SuspendVM_Task
+:host_delete:
+  :queue_timeout: 20.minutes
+:host_introspect:
+  :queue_timeout: 20.minutes
+:host_manageable:
+  :queue_timeout: 20.minutes
+:host_provide:
+  :queue_timeout: 20.minutes
 :host_scan:
   :queue_timeout: 20.minutes
+:host_start:
+  :queue_timeout: 20.minutes
+:host_stop:
+  :queue_timeout: 20.minutes
```

Comment:
I'm going to merge this PR as is, because this would have to be corrected in a follow up anyway.  @blomquisg Can you please come up with a plan to move these into a subkey or something?